### ISSUE TITLE
Confusing about the normalization method.

### ISSUE THREAD

**Issue by mchengny** [MVIG-SJTU/AlphaPose#472]
Hi guys, thanks for your efforts to this project. Here I have a little confusion about the pre-processing method.

In the 740th-line of the file "dataloader.py",  a function named "crop_from_dets" is used to extract the ROI from the original image, in this function you implement a  mean subtraction for each color channel. 

 I see in this part, the order of means  is [0.406, 0.457, 0.480]. While according to [pytorch](https://pytorch.org/docs/master/torchvision/models.html), this is an order for BGR image, and your model takes RGB image as the input.

I wonder that whehther is a detailed trick or just meaningless details. I am looking forawrd to your reply.

Thanks again.

**Comment by mchengny** [comment]
An other question I hope to know is:

I thoroughly read your codes, and try to find the details in each step. Here I just find that your preprocessing method has implemented a mean substraction for the input image, but I didn't see the  division of std (which is usually a part of normalization). 

I hope to know whether my understanding is correct? Or maybe I ignored it during reading.

**Comment by Fang-Haoshu** [comment]
Hi, the means just align with our training process.
The std is not needed according to our experiments.
Thanks.

**Comment by Fang-Haoshu** [comment]
BTW, we have just released the new master branch which is also PyTorch based and easier to train. You can also take a look.
Regards.
